visidata/freqtbl.py

```diff
@@ -26,6 +26,8 @@ def calcValue(col, row):
 
 
 def makeFreqTable(sheet, *groupByCols):
+    if not any(groupByCols):
```

Comment:
If we're just checking that the set is non-empty, we don't need the `any` (I'll fix it).

Reply:
`any` catches the case of `F` on a 0-column sheet, where `cursorCol` is `None` and `groupByCols` is `(None,)`

Reply:
Ah, okay, thanks for clarifying, I'll revert then.  But, thinking out loud here, maybe we should consider making the cursorCol property fail instead of returning None.